ci-operator/step-registry/ipi/install/post/featureset/ipi-install-post-featureset-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: ipi-install-post-featureset
+  from_image:
+    namespace: ocp
+    name: "4.14"
+    tag: upi-installer
+  commands: ipi-install-post-featureset-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+    - name: POST_FEATURE_SET
```

Comment:
How about keeping `FEATURE_SET` as day-0 install.

Reply:
If using same env as day-0, this will also be configured in install-config via step `ipi-conf`, which is not expected.

Reply:
ah, good point, you are right.